frontend/src/components/CoronaActivitiesComponents/ProvinceIncidenceBubble.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from 'styled-components/macro'
+import {getColorCode} from "../../service/incidenceColorService";
+
+export default function ProvinceIncidenceBubble({incidenceValue, incidenceLevel, province}) {
+    return (
+        <BubbleWrapper>
+            <p>{province}</p>
+            <Bubble coronaColor={getColorCode(incidenceLevel)}>{incidenceValue}</Bubble>
+        </BubbleWrapper>
+    )
+}
+
+const BubbleWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--color-cultured);
+  box-shadow: 2px 2px 2px var(--color-silver);
+  border-radius: 3px;
```

Comment:
These rules are the same as in `ProviceCapitalWeather`, maybe  extract the common styles to a component?